### DISCUSSION TITLE
Security issue in 4.6.3 after upgrading it from 4.5.1

### DISCUSSION THREAD

**Discussion by tabhus** [micronaut-projects/micronaut-platform#1714] [General]
I'm encountering an issue with Micronaut security in 4.6.3. I have security enabled in my application.properties file:

micronaut.security.enabled=true

However, when I try to access a controller method annotated with @Secured(SecurityRule.IS_ANONYMOUS), I receive an "Unauthorized exception." Here's the relevant code snippet:

@Controller("/api/v1")
public class MyController {

    @Get("/anonymous-endpoint")
    @Secured(SecurityRule.IS_ANONYMOUS)
    public String getAnonymousData() {
        return "This data is accessible to anonymous users.";
    }
}

**Comment by sdelamo** [comment]
You don't need to specify `micronaut.security.enabled=true`. It is enabled by default since Micronaut Security 2.0.

do you have a sample app which reproduces the issue?.
